packages/platform-sdk/test/integrationTests/shopping-list/shopping-list.test.ts

```diff
@@ -0,0 +1,223 @@
+import { randomUUID } from 'crypto'
+import { apiRoot } from '../test-utils'
+import {
+  CustomerResourceIdentifier,
+  ShoppingListDraft,
+  TextLineItemDraft,
+} from '../../../src'
+import {
+  createCustomer,
+  createCustomerDraft,
+  deleteCustomer,
+  deleteCustomerFromUpdatableObject,
+} from '../customer/customer-fixture'
+import {
+  createCustomerGroup,
+  deleteCustomerGroup,
+} from '../customer-group/customer-group-fixture'
+import {
+  createShoppingList,
+  createShoppingListDraft,
+  deleteShoppingList,
+} from './shopping-list-fixture'
+
+describe('testing shopping list API calls', () => {
+  it('should create and delete a shopping list by ID', async () => {
+    const customerGroup = await createCustomerGroup()
```

Comment:
here is the same like previous comment: "Leaving these methods here to show the customer step-by-step how to create/delete the final object"